src/workspace-tree/bazel_workspace_folder_tree_item.ts

```diff
@@ -141,7 +143,7 @@ export class BazelWorkspaceFolderTreeItem implements IBazelTreeItem {
   /**
    * Returns a promise for an array of tree items representing build packages.
    */
-  private async getPackages(): Promise<BazelPackageTreeItem[]> {
+  private async getPackages(): Promise<IBazelTreeItem[]> {
```

Comment:
Nit: for future us, we should probably change the name of this method now to `getTopLevelItems` (and update the call site in `getChildren`) so it's more accurate.

Reply:
`getTopLevelItems` isn't really right either, _top_ implies (to me aleast) things at the WORKSPACE level.  This is getting the packages and items from the opened directory.

Reply:
Well, the function is kind of odd in the sense that it gets *all* the packages but then builds the whole tree and only returns the top-level items.

`buildPackageTreeAndReturnTopLevelPackagesAndTargets`? 😅

Reply:
That better?

Reply:
WFM